packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/context/index.tsx

```diff
@@ -459,8 +504,30 @@ export const InheritanceGoldPlanPurchaseDialogProvider: FC<
           return true;
         },
       },
+      [tabIndicies.message.tabNumber]: {
+        [tabIndicies.message.dialogs.video]: () => {
+          if (!haveExecutor) {
+            goTo(
+              tabIndicies.nominieeAndExecutor.tabNumber,
+              tabIndicies.nominieeAndExecutor.dialogs.executorSelect,
+            );
+            return true;
+          }
+          return false;
+        },
+        [tabIndicies.message.dialogs.personalMessageInput]: () => {
+          if (!haveExecutor) {
+            goTo(
+              tabIndicies.nominieeAndExecutor.tabNumber,
+              tabIndicies.nominieeAndExecutor.dialogs.executorSelect,
+            );
+            return true;
+          }
+          return false;
+        },
```

Comment:
I don't think this is required, back on message input should just lead back to the video